src/js/video.js

```diff
@@ -89,6 +89,35 @@ var videojs = function(id, options, ready){
   return tag['player'] || new Player(tag, options, ready);
 };
 
+/**
+ * Expose private objects on the global videojs namespace using a Proxy if
+ * supported. Browsers that support Proxy will log a deprecation warning for
+ * directly accessing or modifying a deprecated object.
+ *
+ * @private
+ * @param {String} name The key name for the deprecated property.
+ * @param {Object} target The target object.
+ * @param {Object} messages String messages to display from a Proxy, mapping to get/set operations.
+ * @return {Object} A Proxy if supported or the `target` argument.
+ */
+const createDeprecationProxy = (name, target, messages) => {
```

Comment:
:+1: on both.

The `name` argument is a holdover from the original implementation, which generated the log messages. Since that doesn't happen anymore, it's superfluous and your suggestion is better!